docs/sources/slack/example-api-responses/original/discovery-conversations-history.json

```diff
@@ -196,7 +196,107 @@
           ]
         }
       ]
+    },
+    {
+      "user": "USLACKBOT",
+      "channel": "D075J2RKK4Y",
+      "room": {
+        "id": "R07BNSQTZSP",
+        "name": "huddle test topic",
+        "media_server": "",
+        "created_by": "U02K5LRARED",
+        "date_start": 1720542372,
+        "date_end": 1720542756,
+        "participants": [],
+        "participant_history": [
+          "U02K5LRARED",
+          "U074XMEDGUU"
+        ],
+        "participants_events": {
+          "U02K5LRARED": {
```

Comment:
Redacted as currently we don't have support for transforming keys, like *U02K5LRARED* to its pseudonymized identity.
Also, with `participants_history` is enough for the Worklytics app side